### PR TITLE
Suunto App Modifications per code review

### PR DESCRIPTION
**Pull request by FJBDev**
See this previous [PR](https://github.com/wolfgang-ch/mytourbook/pull/299) and this [PR](https://github.com/wolfgang-ch/mytourbook/pull/300)

FIT SDK 21.47

### CODE REVIEW

**Review by wolfgang-ch — COMMENTED** [review]
This PR is now merged into main but I'm not very happy with the text key names

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java [hunk 1]
```diff
@@ -173,15 +174,15 @@ private void createUI_20_AccountInformation(final Composite parent) {
       {
```
Comment (wolfgang-ch) [inline bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java:174]: To have a common UI between all cloud providers, I'm missing a web page url when a user is not yet authorized...

mt-cloud-strava

... like in the other cloud pref pages

mt-cloud-other

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java [hunk 2]
```diff
@@ -21,8 +21,10 @@
 
    private static final String BUNDLE_NAME = "net.tourbook.cloud.messages";   //$NON-NLS-1$
 
-   public static String        Html_CloseBrowser_Text;
+   public static String        Icon__Check;
+   public static String        Icon__Hourglass;
```
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java:25]: I thought it was supposed to be like all other images. See [this one](https://github.com/wolfgang-ch/mytourbook/blob/73582970e988812e11dd332405c066fa4acc33de/bundles/net.tourbook/src/de/byteholder/geoclipse/messages_de.properties#L257) for example.

Should I just then put the string itself ?

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java:174]: I will add that in another PR

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java [hunk 3]
```diff
@@ -21,8 +21,10 @@
 
    private static final String BUNDLE_NAME = "net.tourbook.cloud.messages";   //$NON-NLS-1$
```
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java:22]: I will update the naming in my next PR

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (wolfgang-ch) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java:25]: Symbols/icons are defined in

https://github.com/wolfgang-ch/mytourbook/blob/73582970e988812e11dd332405c066fa4acc33de/bundles/net.tourbook.common/src/net/tourbook/common/UI.java#L147-L198

and images are only created when symbols are not available.

I'm currently not happy that all images are defined in messages.properties and these entries are duplicated in 99.99% cases for all languages. Currently there are only a few images which contain a alphabetical character which is translated. But I've not yet an idea how to solve this, and it's also a lot of work.

PS: Do not complain that `SYMBOL_MIN` and `SYMBOL_MAX` have the wrong sorting order, when e.g. 2 fields belong together, then (not always) I sort them how they are normally spoken. Up to 99% you talk about min/max values and not max/min values

[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java:25]: >I'm currently not happy that all images are defined in messages.properties and these entries are duplicated in 99.99% cases for all languages

Would it help if you were to define "clean-up" tasks such as this. I would be happy to help with those tasks if you feel it is important to do.
Example:
"- Remove all the translated images"

and maybe you have other lists of tasks to do for the code

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java (hunk 3 above)
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java:22]: Fixed in [PR](https://github.com/wolfgang-ch/mytourbook/pull/305)

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java (hunk 1 above)
Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java:174]: Fixed in [PR](https://github.com/wolfgang-ch/mytourbook/pull/305)

File: bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (FJBDev) [reply bundles/net.tourbook.cloud/src/net/tourbook/cloud/Messages.java:25]: Fixed in [PR](https://github.com/wolfgang-ch/mytourbook/pull/305)

**Pull request merged**